cmd/antrea-agent/options.go

```diff
@@ -453,11 +467,10 @@ func (o *Options) validateK8sNodeOptions() error {
 			}
 		}
 	}
-	if (features.DefaultFeatureGate.Enabled(features.Multicluster) || o.config.Multicluster.Enable) &&
-		encapMode != config.TrafficEncapModeEncap {
-		// Only Encap mode is supported for Multi-cluster feature.
-		return fmt.Errorf("Multicluster is only applicable to the %s mode", config.TrafficEncapModeEncap)
+	if err := o.validateMulticlusterConfig(encapMode); err != nil {
+		return fmt.Errorf("failed to validate Multicluster config: %v", err)
```

Comment:
Just return the original error, which already contains enough information.

Reply:
Done.